src/chi-vue/src/components/data-table/DataTable.tsx

```diff
@@ -984,9 +1042,14 @@ export default class DataTable extends Vue {
       columnHeadCell,
       columnHeadSortButton;
 
+    if ((element && element.classList.contains(INFO_ICON_CLASS)) || element.querySelector(`.${INFO_ICON_CLASS}`)) {
```

Comment:
Why is this necessary?

Reply:
these check that the element that has been clicked isn't the icon-info button or its icon, it's needed because otherwise every time you click on the info button the column gets sorted, and we only want the popover to open (without sorting).